lib/drivers/driver.js

```diff
@@ -163,4 +163,42 @@ Driver.prototype.push = function (opts) {};
  */
 Driver.prototype.status = function (opts) {};
 
+/**
+ * Track local files into a git repository. Performs a post request to /track
+ * endpoint with a JSON payload in the form of:
+ *
+ *
+ *          {
+ *            "message": "Commit message",
+ *            "files": [{
+ *                "name": "README.md",
+ *                "status": "modified"
+ *            }, {
+ *                "name": "foobar.md",
+ *                "status": "untracked"
+ *            }],
+ *
+ *            "author": {
+ *                "name": "John Doe",
+ *                "email": "johndoe@example.com",
+ *                "date": "Mon Nov 07 2016 16:40:35 GMT+0100 (CET)"
+ *            },
+ *
+ *            "committer": {
+ *                "name": "John Doe",
+ *                "email": "johndoe@example.com",
+ *                "date": "Mon Nov 07 2016 16:40:35 GMT+0100 (CET)"
+ *            }
+ *        }
+ *
+ * @param {String} opts.message List of files with name and status.
```

Comment:
"commit message" ?
If it is optional, you can put it between brackets `[opts.message]` or indicate it is `(optional)` as for `opts.author` for example.